ios/RNCallKeep/RNCallKeep.m

```diff
@@ -570,20 +585,6 @@ + (NSArray *) getAudioInputs
 
     AVAudioSession* myAudioSession = [AVAudioSession sharedInstance];
 
```

Comment:
Tested and I noticed if we remove the `setCategory` from the `getAudioInputs`, the availableInputs are not up-to-date, I mean if in the meantime you remove a connected bluetooth device, the availableInputs are not updated

Reply:
I'm using the `getAudioRoutes`